papers/project_jupyter/paper.rst

```diff
@@ -666,21 +682,34 @@ is avoided. In addition,
 a BinderHub deployment can efficiently use the resources available to it in
 order to avoid over-provisioning cloud resources as much as possible.
 
-Currently, the hosting bill for `mybinder.org <https://mybinder.org>`_ runs at a cost of several hundred dollars per
-day. At roughly 50,000 users per week, this comes out to around $\frac{220 \times30}{50000} \approx 13$cents per
+Currently, the hosting bill for `mybinder.org <https://mybinder.org>`_ runs at a cost of
+around $180 per day and around 7,000 users per day. This
+comes out to around :math:`\frac{180 \times30}{7000 \times30} \approx 3` cents per
 user. The `mybinder.org <https://mybinder.org>`_ team publishes its daily hosting costs in a public
 repository on GitHub :cite:`JupyterHub2018-ek`.
 It hopes that this serves to encourage other organizations to deploy BinderHub
 for their own purposes, since it is possible to do so in a cost-effective
 manner.
 
+Finally, because Kubernetes is an open source system for managing containers, it has been deployed
+on a number of cloud providers as well as on self-owned hardware and virtual machines.
+While `mybinder.org <https://mybinder.org>`_ currently runs on the Google Cloud Platform,
+a BinderHub can run on any setup that runs on top of Kubernetes. This flexibility
```

Comment:
"a BinderHub can run on any setup that runs on top of Kubernetes" -> "a BinderHub can run on top of any Kubernetes cluster that supports XXX"? We need some kind of caveat here as not every k8s cluster is equal to every other in terms of functionality :-/

Reply:
ah this is a good point - do you wanna have a go at updating the text, or want me to do it?

Reply:
I'd be happy if you did it, otherwise it won't get done for a while by me :-/

Reply:
I added a quick sentence mentioning hardware, nothing too strong though